fastapi_app/tests/test_hello.py

```diff
@@ -0,0 +1,9 @@
+import httpx
+import pytest
+
+@pytest.mark.asyncio
+async def test_hello():
+    async with httpx.AsyncClient() as client:
+        response = await client.get("http://localhost:8001/hello")
```

Comment:
If the FastAPI app will be mounted at a sub-path (e.g., `/api/fastapi/hello`), update this test to reflect the correct endpoint.